DPRHTML/index.html

```diff
@@ -126,12 +126,17 @@
       var location = document.location.href;
       if (location.indexOf('?') > -1) {
         loadSidebarDivs();
+        if(location.indexOf('analysis')>-1){
+          x= new URL(location);
+          console.log("found");
```

Comment:
Is it possible to have analysis param along with feature=search/dictionary? asking since i am not sure and if that is the case, we'll break those scenarios.